src/components/frame/v5/pages/MembersPage/partials/SubNavigation/SubNavigation.tsx

```diff
@@ -46,7 +47,11 @@ const SubNavigation: FC = () => {
         </button>
       </li>
       <li>
-        <button type="button" className={styles.navigationButton}>
+        <button
```

Comment:
I think we can create a new variant of the button because we will use it in this and the third stage a few times.

Also, I am not sure if it should be placed exactly on MembersPage. A similar Navigation will be used also in the third stage. I think we should create a more reusable component that we could use in the future.

Reply:
New button variant added, I think this can stay in MembersPage because this menu will be used only here. PopoverBase is component that we are reusing.